### ISSUE TITLE
Improve registration quality

### ISSUE THREAD

**Issue by JdbermeoUZH** [JdbermeoUZH/MastersThesisUIASegmentation#42]
DoD:
Checked different hyperparameters of the algorithm such as:
 - Image with higher resolutions
 - Different initialization: centered
 - More iterations or smaller learning rate
